src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt

```diff
@@ -213,11 +211,32 @@ open class LatexAnnotator : Annotator {
     /**
      * Annotates the contents of the given parameter with the given style.
      */
+    @Suppress("USELESS_CAST")
     private fun AnnotationHolder.annotateRequiredParameter(parameter: LatexRequiredParam, style: TextAttributesKey) {
-        val content = parameter.firstChildOfType(LatexContent::class) ?: return
```

Comment:
This line must have been a mistake, there is never a Content that is a child of a RequiredParam (check `latex.bnf`).
So skip that, just use `LatexRequiredParamContent` as you did. Good find!